packages/react/src/React.js

```diff
@@ -3,6 +3,8 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
```

Comment:
This was the linchpin change that uncovered all of the masked `any` type errors.